hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/NamenodeBeanMetrics.java

```diff
@@ -884,6 +884,11 @@ public int getPendingSPSPaths() {
     return 0;
   }
 
+  @Override
+  public double getReconstructionQueuesInitProgress() {
```

Comment:
Is it `float` enough?

Reply:
Hi Sir, thanks for your suggestion.
I think the reconstructionQueuesInitProgress is enough to use float, here is just to get a percentage progress, maybe don't need higher precision, i will update it later.